### DISCUSSION TITLE
windows系统下，导出后PAG预览工具无法显示已下载字体，请问怎么解决

### DISCUSSION THREAD

**Discussion by EddieHxc** [Tencent/libpag#2339] [General] [answered]
PAG导出后无法正常显示字体，查看本地字体资源是已安装的

**Comment by kevingpqi123** [comment, marked as answer]
可以看下这篇文档：https://pag.art/docs/faq.html#%E9%97%AEpag-%E4%B8%AD%E5%AD%97%E4%BD%93%E5%8A%A0%E8%BD%BD%E7%9A%84%E7%AD%96%E7%95%A5%E6%98%AF%E4%BB%80%E4%B9%88-linux-%E7%AB%AF%E6%96%87%E6%9C%AC%E4%B8%BA%E5%95%A5%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA

**Reply by EddieHxc** [reply]
好的 可能是不同windows系统字体刷新拉去不到吧

**Reply by EddieHxc** [reply]
请问关于BMP视频导出呢，用最新版本的工具 只要添加BMP这个形式 就会导出失败

**Reply by kevingpqi123** [reply]
> 好的 可能是不同windows系统字体刷新拉去不到吧

和这个关系不大，桌面预览工具并不能保证匹配字体，匹配字体只能通过调用 API 接口注册

**Reply by kevingpqi123** [reply]
> 请问关于BMP视频导出呢，用最新版本的工具 只要添加BMP这个形式 就会导出失败

我们会抽时间统一看下，可以先使用历史版本：https://pag.art/docs/pag-download.html
